frontend/src/exploration/previews/LayoutCard.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  ListItem,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import React from 'react';
+import { createStyles } from '@material-ui/core/styles';
+import LayoutDefinition from './LayoutDefinition';
+
+const useStyle = makeStyles(() =>
+  createStyles({
+    card: {
+      width: '100%',
+    },
+    media: {
+      height: 0,
+      paddingTop: '56.25%', // 16:9
+    },
+  })
+);
+
+/**
+ * A Preview of a Layout that routes to the tab path from the input parameter.
+ * @param layout
+ */
+function LayoutCard(layout: LayoutDefinition): JSX.Element {
+  const classes = useStyle();
+
+  const { path, filename, description } = layout;
+
+  const imagePath = '/exploration-preview/'.concat(filename, '.png');
+
+  return (
+    <a href={path} style={{ textDecoration: 'none' }}>
```

Comment:
Don't use href because that disables advantages of the SPA. https://reactrouter.com/native/api/Link